### ISSUE TITLE
Video ads is not showing on mobile devices !

### ISSUE THREAD

**Issue by quangtan13** [jwplayer/jwdeveloper-demos#479]
hi team!
video ads (VAST) not working, when i run your demo website on android devices. it's working well on PC or tablet only.
pls fix it ! thanks

**Comment by lagardereactive** [comment]
do you have some news please ? 
Thank you !

**Comment by quangtan13** [comment]
No, i cant fix it, on safari and android browser (mobile devicer) can not
show ads (tested on demo site and my site)

Vào 21:02, T.3, 19 Th3, 2019 lagardereactive <notifications@github.com đã
viết:

**Comment by cmifrance** [comment]
but to Chrome and Firefox it works ?

**Comment by cmifrance** [comment]
do you have an url to see it please ?

**Comment by quangtan13** [comment]
Tutorials.youphptube.com/video/how-set-a-video-gellary-and-set-video-ads

Pls open it on android browser , u will see that

**Comment by cmifrance** [comment]
on safari mobile it's works and the same to the android default browser it works
![55455299_573419179827841_1356828439943839744_n](https://user-images.githubusercontent.com/30118156/54619877-98f73180-4a65-11e9-8328-acadb0a1d933.jpg)

**Comment by quangtan13** [comment]
Now i show u the problem by my video.
First i use android browser, it not works.then i open chrome, it works.


Vào 22:39, T.3, 19 Th3, 2019 CMI France <notifications@github.com đã viết:

**Comment by cmifrance** [comment]
i have use the android browser for this test
